### ISSUE TITLE
Use alt text to describe the image.

### ISSUE THREAD

**Issue by discospiff** [mikeseitz1/IS7012#5]
Use alt text to give an accurate, yet concise description of what the image looks like, for the benefit of users who cannot see the image.

https://github.com/mikeseitz1/IS7012/blob/e9625df4fcf093557a0af2d6c3718b9457d73667/Week2/RecruitCatSeitzme/Pages/Index.cshtml#L12
